oob-messages/README.md

```diff
@@ -0,0 +1,44 @@
+## oob-messages
+
+A Rust library for implementing out of band messages for DID-based applications.
+
+>Out of band messages (OOB) messages are the initiators of a didcomm communication where by sender provides which identifier in an unencrypted messages (QR-code or Invitation link) which the other party can scan with his/her edge device, hence no private information should be send in this way. The protocol used here is the version 2 ```https://didcomm.org/out-of-band/2.0/invitation```
```

Comment:
What you quote should originate from another source verbatim.
```suggestion
>Out of band messages (OOB) messages are the initiators of a didcomm communication where by sender provides which identifier in an unencrypted messages (QR-code or Invitation link) which the other party can scan with his/her edge device, hence no private information should be send in this way. 

The protocol used here is the version 2 ```https://didcomm.org/out-of-band/2.0/invitation```
```

Reply:
ok I've done the changes